src/components/ComboBox/ComboBox.js

```diff
@@ -26,6 +22,28 @@ const getInputValue = (props, state) => {
   return state.inputValue || '';
 };
 
+const findHighlightedIndex = ({ items, itemToString }, inputValue) => {
+  if (!inputValue) {
+    return -1;
+  }
+
+  let i = 0;
```

Comment:
What do you think about a `for` loop variant of this:

```suggestion
  const searchValue = inputValue.toLowerCase();
  
  for (let i = 0; i < items.length; i++) {
    const item = itemToString(items[i]).toLowerCase();
    if (item.indexOf(searchValue) !== -1) {
      return i;
    }
  }

  return -1;
```

This can help drop our usage of `includes`, as well.

Reply:
Better!
Thanks.